bioModels/matRad_BiologicalModel.m

```diff
@@ -272,10 +275,48 @@
                                 matRad_cfg.dispWarning(['matRad: Invalid biological optimization quantity: ' this.quantityOpt  '; using "none" instead.']);
                                 this.quantityOpt = 'physicalDose';
                         end
+
+                    case {'brachy'} 
+                        
+                        setDefaultValues = false;
+                        switch this.quantityOpt
+
+                            case {'physicalDose'}
+                                if strcmp(this.model,'none')
+                                    boolCHECK        = true;
+                                    this.bioOpt      = false;
+                                    this.quantityVis = 'physicalDose';
+                                else
+                                    setDefaultValues = true;
+                                end
+
+                            case {'RBExD'}
+                                if sum(strcmp(this.model,{'constRBE', 'none'})) == 1
+                                    this.RBE         = this.constRBE_photons;
+                                    boolCHECK        = true;
+                                    this.bioOpt      = false;
+                                    this.quantityVis = 'RBExD';
+                                else
+                                    setDefaultValues = true;
+                                end
+
+                            case {'effect'}
+                                if strcmp( this.model,'none')
+                                    boolCHECK        = true;
+                                    this.bioOpt      = true;
+                                    this.quantityVis = 'RBExD';
+                                else
+                                    setDefaultValues = true;
+                                end
+
+                            otherwise
+                                matRad_cfg.dispWarning('matRad: Invalid biological optimization quantity: %s; using physical dose instead.',this.quantityOpt);
+                                this.quantityOpt = 'physicalDose';
+                        end
                                       
                     otherwise
                         matRad_cfg.dispWarning(['matRad: Invalid biological radiation mode: ' this.radiationMode  '; using photons instead.']);
-                        this.radiationMode = 'photons';
+                        this.radiationMode = 'brachy';
```

Comment:
whats going on here ? 'brachy' as default radiation mode ?

Reply:
this is something i changed at the start because we have on the example the biological models and shouldn't it be as a radiation mode ? If the example doesn't need the biological models then we don't need it there. I just assumed because all the examples have the biological model called, the brachy example won't be an exception hence why i added it there as a radiation mode. I don't know if i am making sense, feel free to correct me.

Reply:
I think the point is that it should not default to "brachy" if the radiationMode is not recognized.